lib/identity_store.js

```diff
@@ -180,6 +180,29 @@ IdentityStore.prototype = {
             });
     },
 
+    /**:IdentityStore.optin(identity)
+    Posts an optin to the identity store optout endpoint
+
+    :param object identity:
+      Gets passed through directly to service. Consult service docs/protocol.
+      An example of identity object:
+        {
+            "identity": "uuid",
+            "address_type": "msisdn",
+            "address": "+27845000000"
+        }       
+    */
+    optin: function(identity) {
```

Comment:
can we make the 3 params obvious to the user by splitting them out and using a `payload` builder like the other functions?